servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -147,6 +151,86 @@ def get_job(id, **kwargs):
                                   parent_job_id=job.get('parentWorkflowId')))
 
 
+@requires_auth
+def get_task_attempts(id, task, **kwargs):
+    """
+    Query for attempt metadata for a specified job task
+
+    :param id: Job ID
+    :type id: str
+
+    :param task: Task Name
+    :type task: str
+
+    :rtype: JobAttemptsResponse
+    """
+
+    include_keys = ('attempt', 'callCaching:hit',
+                    'callCaching:effectiveCallCachingMode', 'callRoot', 'end',
+                    'executionStatus', 'failures', 'inputs', 'jobId', 'labels',
+                    'outputs', 'shardIndex', 'start', 'stderr', 'stdout')
+
+    url = '{cromwell_url}/{id}/metadata?{query}'.format(
+        cromwell_url=_get_base_url(),
+        id=id,
+        query='includeKey=' + '&includeKey='.join(include_keys))
+    response = requests.get(url,
+                            auth=kwargs.get('auth'),
+                            headers=kwargs.get('auth_headers'))
+
+    if response.status_code != 200:
+        handle_error(response)
+
+    job = response.json()
+
+    attempts = [
+        _convert_to_attempt(call) for call in job.get('calls').get(task)
+    ]
+    return JobAttemptsResponse(attempts=attempts)
+
+
+@requires_auth
+def get_shard_attempts(id, task, index, **kwargs):
+    """
+    Query for attempt metadata for a specified job task shard
+
+    :param id: Job ID
+    :type id: str
+
+    :param task: Task Name
+    :type task: str
+
+    :param index: Shard Index
+    :type index: str
+
+    :rtype: JobAttemptsResponse
+    """
+
+    include_keys = ('attempt', 'callCaching:hit',
+                    'callCaching:effectiveCallCachingMode', 'callRoot', 'end',
+                    'executionStatus', 'failures', 'inputs', 'labels',
+                    'outputs', 'shardIndex', 'start', 'stderr', 'stdout')
```

Comment:
this tuple seems to be used in more than one spot, can it be extracted to a common place?